### DISCUSSION TITLE
please add these quotes

### DISCUSSION THREAD

**Discussion by lenincritus** [monkeytypegame/monkeytype#1164] [Ideas] [answered]
i tried to git push but somehow github wont so i am posting the quotes along with code so you wont have to type them, please merger these:

     },
     {
       "text": "Do not pity the dead, Harry. Pity the living, and, above all those who live without love.",
       "source": "Albus Dumbledore",
       "length": 89,
       "id": 5447
     },
     {
       "text": "It does not do well to dwell on dreams and forget to live.",
       "source": "Albus Dumbledore",
       "length": 58,
       "id": 5448
     },
     {
       "text": "Have you ever had a dream, Neo, that you seemed so sure it was real? But if were unable to wake up from that dream, how would you tell the difference between the dream world & the real world?",
       "source": "The Matrix",
       "length": 191,
       "id": 5449
     },
     {
       "text": "If real is what you can feel, smell, taste and see, then 'real' is simply electrical signals interpreted by your brain.",
       "source": "The Matrix",
       "length": 119,
       "id": 5450

**Comment by unrealapex** [comment]
Quote submissions are available on the [Monkeytype Discord server](https://www.discord.gg/monkeytype), in the `#quote-submission channel`.

**Reply by Miodec** [reply]
Honestly when it comes to adding many quotes Discord is the worst way of doing it - very slow to add them one by one

**Comment by Miodec** [comment, marked as answer]
Added
